src/Microsoft.ML.Transforms/RandomFourierFeaturizing.cs

```diff
@@ -572,8 +565,8 @@ private ValueGetter<VBuffer<float>> GetterFromFloatType(IRow input, int iinfo)
                 var getSrc = input.GetGetter<float>(_srcCols[iinfo]);
                 var src = default(float);
 
-                var featuresAligned = new AlignedArray(RoundUp(1, _cfltAlign), CpuMathUtils.GetVectorAlignment());
-                var productAligned = new AlignedArray(RoundUp(_parent._transformInfos[iinfo].NewDim, _cfltAlign), CpuMathUtils.GetVectorAlignment());
+                var featuresAligned = new float[4];
```

Comment:
I'm not sure `aligned` is a good name for this anymore (given that the array is not guaranteed to be or stay aligned).